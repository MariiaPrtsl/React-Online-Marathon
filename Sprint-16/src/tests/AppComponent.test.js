import renderer from 'react-test-renderer'
import App from '../components/App/index.js'

// TODO: Your test need to be here instead of fake tests

describe("test App Component", () => {
  it("matches the snapshot", () => {
      const wrapper = renderer.create(<App/>).toJSON();
      expect(wrapper).toMatchSnapshot();
  });
});