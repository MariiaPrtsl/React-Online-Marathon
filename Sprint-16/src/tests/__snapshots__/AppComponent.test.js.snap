// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render App Component matches the snapshot 1`] = `
<div
  className="App"
>
  <h1>
    Hello World!
  </h1>
  <div>
    <h3>
      GitHub User Info
    </h3>
    <ul />
  </div>
  <div>
    <h3>
      GitHub User Info
    </h3>
    <ul />
  </div>
</div>
`;

exports[`test App Component matches the snapshot 1`] = `
<div
  className="App"
>
  <h1>
    Hello World!
  </h1>
  <div>
    <h3>
      GitHub User Info
    </h3>
    <ul />
  </div>
  <div>
    <h3>
      GitHub User Info
    </h3>
    <ul />
  </div>
</div>
`;
